pgpy/packet/fields.py

```diff
@@ -783,12 +793,17 @@ def __init__(self):
         # iterated
         self.count = 0
 
+        # GNU extension smartcard
+        self.scserial = None
+
     def __bytearray__(self):
         _bytes = bytearray()
         _bytes.append(self.usage)
         if bool(self):
             _bytes.append(self.encalg)
             _bytes.append(self.specifier)
+            if self.specifier > String2KeyType.Iterated:
```

Comment:
this should be `if self.specifier == String2KeyType.GNUExtension`; we don't want to be unintentionally overly greedy with other experimental extensions we don't yet know about